src/Sylius/Behat/Context/Ui/Admin/ManagingTaxCategoryContext.php

```diff
@@ -26,14 +27,19 @@
     const RESOURCE_NAME = 'tax_category';
 
     /**
-     * @var SharedStorageInterface
+     * @var IndexPageInterface
      */
-    private $sharedStorage;
+    private $taxCategoryIndexPage;
```

Comment:
taxCategory prefix is redundant when we are in ManagingTaxCategoryContext. This applies to all other contexts I assume. We should remove it, wdyt?